### ISSUE TITLE
MPV outputs blue image instead of the video when CSGO is opened in another window, bug does not occur if MPV is opened before launching CSGO, only after, and it is random, (logs and screenshots included!)

### ISSUE THREAD

**Issue by bobafettyyy** [mpv-player/mpv#9271]
### Important Information

Provide following Information:
mpv-x86_64-20211003-git-5304e9f
Windows 10
source: sourceforge.ne
this problem was active in every update of last year
screens: screenshot of what the video looks like https://i.imgur.com/YalImYh.jpg , and the screenshot mpv creates when taking a screenshot https://i.imgur.com/o2sOKIz.jpg

### Reproduction steps

so this bug only appears for me when i'm running CSGO at the same time and i'm trying to load MPV to play something, and while its occurrence is random, MPV plays the audio but not video, the video doesn't appear and instead a constant blue color appears as seen in this screenshot https://i.imgur.com/YalImYh.jpg ,

NOTE: This bug only appeared when i launched MPV with custom config, when I launched it with the no config argument, the bug didn't appear
here is my mpv.conf file: https://0x0.st/-g9J.conf

### Log file
logs: https://0x0.st/-g9W.txt

logs with the --no-config argument:
https://0x0.st/-g94.txt

### Sample files
this bug persists with any video file regardless of the codec/format (it appeared with both HEVC and AVC files), so no specific sample files are needed to recreate it

**Comment by haasn** [comment]
Can you retry with a version of libplacebo that includes 9c0007a4428a8b9c753fe8300ae8b331b0bb896e?

**Comment by bobafettyyy** [comment]
Here is a screenshot of GPU memory usage while this bug was present AND CSGO was opened, https://0x0.st/-Ep3.jpg , there is still 5GB of video memory left, so i think that we can rule out GPU running out of space, also @haasn could you tell me how to do that, I'm using Windows 10 if that helps?

**Comment by bobafettyyy** [comment]
> Does it happen right away for you or only after a bit of playback? If the latter, it might be a duplicate of #9248.
if I launch an MPV window and drag and drop a video or a collection of videos before launching the game, the video plays fine, but if i import (drag and drop) a new video/playlist in the existing or a new window of MPV after launching the game then the video outputs only blue

**Comment by Dudemanguy** [comment]
Possibly fixed by https://github.com/haasn/libplacebo/commit/627951af3dea62061c41fdb4c7ee095f483476ca. If not, please up a new issue.
